staging/src/k8s.io/apiserver/pkg/endpoints/filters/metrics_test.go

```diff
@@ -17,131 +17,143 @@ limitations under the License.
 package filters
```

Comment:
Shouldn't this be moved along with metrics.go? It looks like you added a second metrics_test.go, so maybe merge the two?

Reply:
I had to split-out tests that use the "union" package to avoid circular dependency between authenticator and filter packages. 
An alternative could be to create a separate testing package.

Reply:
I see, in that case I'd prefer one of these options:
Create a `staging/src/k8s.io/apiserver/pkg/authentication/authenticator/metrics` package
**or**
Move this test to the union package

Reply:
Moved into the union package.